internal/cli/install.go

```diff
@@ -375,9 +367,17 @@ func (c *InstallCommand) Run(args []string) int {
 	s.Update("Server installed and configured!")
 	s.Done()
 
+	// NOTE(briancain): Right now we rely on the fact that only K8S returns a bootstrap
+	// token from the installation interface helper. Since we know helm sets up
+	// a bootstrap job AS WELL AS a runner, we don't need to install another runner
+	// or another runner profile.
 	if c.flagRunner {
+		existingODRConfigSetup := false
+		if bootstrapToken != "" {
```

Comment:
Is this supposed to be `token != ""` ?

Reply:
No because `bootstrapToken` is the variable that gets returned from install, where as `token` could of been setup by the CLI bootstrap. See my other comment on the PR about that variable.